composer.json

```diff
@@ -39,9 +39,9 @@
     "require": {
     },
     "require-dev": {
-        "cakephp/cakephp": "~3.0",
+        "cakephp/cakephp": "3.4.*",
         "cakephp/cakephp-codesniffer": "2.*",
-        "phpunit/phpunit": "4.1.*"
+        "phpunit/phpunit": "^5.7|^6.0"
```

Comment:
Change to "<6.0" so that the test suite runs for older CakePHP version on PHP 5.5.